1-Authentication/1-sign-in/AppCreationScripts/AppCreationScripts.md

```diff
@@ -108,8 +108,8 @@ Note that the script will choose the tenant in which to create the applications,
 
   if you want to create the apps in a particular tenant, you can use the following option:
   
-- Open the [Azure portal](https://portal.azure.com)
-- Select the Azure Active directory you are interested in (in the combo-box below your name on the top right of the browser window)
+- Open the [Microsoft Entra admin center](https://entra.microsoft.com)
```

Comment:
portal.azure.com still works right? I did not see comms go where this is not the case.

Reply:
It does still work but we want customers to start going straight to Microsoft Entra Admin Center when they want to use Microsoft Entra ID since it has replaced Azure Active Directory. I will gladly elaborate on this if you'd like.